part2/bowling/bowling.rb

```diff
@@ -0,0 +1,64 @@
+class Game
+  def initialize
+    @rolls = [] 
+    @rolls_index_position = 0
+    @score = 0   
+    @frame = 1
+  end
+
+  def roll(pins)
+    @rolls << pins
+  end
+
+  def score
+    calculate_total_score
+    @score
+  end
+
+  def update_roll_position(number)
+    @rolls_index_position += number
+  end
+
+  def is_a_strike?
+    @rolls[@rolls_index_position] === 10
```

Comment:
Not sure about `===` in ruby? But if the tests are passing I guess this is valid syntax?